habitat/tasks/rearrange/rearrange_sim.py

```diff
@@ -34,9 +40,16 @@ def __init__(self, config: Config):
         super().__init__(config)
 
         agent_config = self.habitat_config
-        self.navmesh_settings = get_nav_mesh_settings(self._get_agent_config())
+
+        agent_cfg = self._get_agent_config()
+
+        self.navmesh_settings = NavMeshSettings()
+        self.navmesh_settings.set_defaults()
+        self.navmesh_settings.agent_radius = agent_cfg.RADIUS
+        self.navmesh_settings.agent_height = agent_cfg.HEIGHT
+        self.navmesh_settings.agent_max_climb = 0.05
```

Comment:
I'm a bit wary of hardcoded NavMesh parameters in a derived object. Should this be a new agent param with defaults instead? If so, this could be moved into a util somewhere instead of explicitly done only for the rearrange_sim.